node/npm_dependencies.ts

```diff
@@ -14,6 +15,51 @@ import { Logger } from './logger.js'
 
 const TYPESCRIPT_EXTENSIONS = new Set(['.ts', '.cts', '.mts'])
 
+/**
+ * Returns the name of the `@types/` package used by a given specifier. Most of
+ * the times this is just the specifier itself, but scoped packages suffer a
+ * transformation (e.g. `@netlify/functions` -> `@types/netlify__functions`).
+ * https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-scoped-packages
+ */
+const inferDefinitelyTypedPackage = (specifier: string) => {
+  if (!specifier.startsWith('@')) return `@types/${specifier}`
+  const [scope, pkg] = specifier.split('/')
+  return `@types/${scope.replace('@', '')}__${pkg}`
+}
+
+/**
+ * Starting from a `package.json` file, this tries detecting a typescript declaration file.
```

Comment:
nit

```suggestion
 * Starting from a `package.json` file, this tries detecting a TypeScript declaration file.
```

Reply:
fixed in https://github.com/netlify/edge-bundler/pull/505/commits/a0a27f0cc7a1beee318a17642450fefe09afcb66